<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>2000</width>
        <height>1500</height>
       </size>
      </property>
      <property name="tabletTracking">
       <bool>true</bool>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
      </property>
      <property name="background">
       <brush brushstyle="CrossPattern">
        <color alpha="255">
         <red>191</red>
         <green>191</green>
         <blue>191</blue>
        </color>
       </brush>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="tabsMovable">
       <bool>false</bool>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>538</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBegin">
    <property name="title">
     <string>begin</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionOpenWith"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="actionTo_PDF"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>编辑</string>
    </property>
    <widget class="QMenu" name="menuSort">
     <property name="title">
      <string>Sort</string>
     </property>
     <addaction name="actionSort_By_Id"/>
     <addaction name="actionSort_By_Name"/>
    </widget>
    <addaction name="actionAdd"/>
    <addaction name="actionDelete"/>
    <addaction name="actionModify"/>
    <addaction name="separator"/>
    <addaction name="menuSort"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionOpen_new_window"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuBegin"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="begin">
   <property name="text">
    <string>开始</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>open</string>
   </property>
   <property name="shortcut">
    <string>Meta+O</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="text">
    <string>Add</string>
   </property>
   <property name="shortcut">
    <string>Meta+N</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>save</string>
   </property>
   <property name="shortcut">
    <string>Meta+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>save as</string>
   </property>
   <property name="shortcut">
    <string>Meta+Shift+S</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Meta+D</string>
   </property>
  </action>
  <action name="actionModify">
   <property name="text">
    <string>Modify</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>about</string>
   </property>
   <property name="shortcut">
    <string>Meta+I</string>
   </property>
  </action>
  <action name="actionOpenWith">
   <property name="text">
    <string>open with</string>
   </property>
  </action>
  <action name="actionSort_By_Id">
   <property name="text">
    <string>Sort By Id</string>
   </property>
  </action>
  <action name="actionSort_By_Name">
   <property name="text">
    <string>Sort By Name</string>
   </property>
  </action>
  <action name="actionOpen_new_window">
   <property name="text">
    <string>New Window</string>
   </property>
   <property name="shortcut">
    <string>Meta+W</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
   <property name="shortcut">
    <string>Meta+P</string>
   </property>
  </action>
  <action name="actionTo_PDF">
   <property name="text">
    <string>To PDF</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionAdd</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_menu_add_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_menu_save_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_menu_about_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSaveAs</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_menu_save_as_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDelete</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_menu_delete_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionModify</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_menu_modify_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_menu_open_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpenWith</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_menu_open_with_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen_new_window</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_menu_OpenNewWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPrint</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_menu_Print_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSort_By_Id</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_menu_sortId()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSort_By_Name</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_menu_sortName()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionTo_PDF</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_menu_ToPDF_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_menu_add_clicked()</slot>
  <slot>on_menu_save_clicked()</slot>
  <slot>on_menu_about_clicked()</slot>
  <slot>on_menu_save_as_clicked()</slot>
  <slot>on_menu_open_clicked()</slot>
  <slot>on_menu_open_with_clicked()</slot>
  <slot>on_menu_delete_clicked()</slot>
  <slot>on_menu_modify_clicked()</slot>
  <slot>on_menu_OpenNewWindow()</slot>
  <slot>on_menu_Print_clicked()</slot>
  <slot>on_menu_sortId()</slot>
  <slot>on_menu_sortName()</slot>
  <slot>on_menu_ToPDF_clicked()</slot>
  <slot>enableTest()</slot>
 </slots>
</ui>
